cgo_go124.go

```diff
@@ -0,0 +1,17 @@
+//go:build go1.24 && !cmd_go_bootstrap
+
+package openssl
+// The following noescape and nocallback directives are used to prevent the Go
```

Comment:
Oops, I didn't quite get the suggestion right. Go format inserts this newline:

```suggestion
package openssl

// The following noescape and nocallback directives are used to prevent the Go
```